### PR TITLE
en, zh: add release notes for v1.1.12

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add release notes for v1.1.12

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/releases/release-1.1.12.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# TiDB Operator 1.1.12 Release Notes
+
+Release date: April 15, 2021
+
+TiDB Operator version: 1.1.12
+
+## New Features
+
+- Support setting customized `ENV` for `Backup` and `Restore` jobs ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
```
Comment (ran-huang) [inline en/releases/release-1.1.12.md:13]: Can you align these notes with [1.2.0-beta.1](https://docs.pingcap.com/tidb-in-kubernetes/dev/release-1.2.0-beta.1)? Most of them are the same.

Comment (csuzhangxc) [reply en/releases/release-1.1.12.md]: done in https://github.com/pingcap/docs-tidb-operator/pull/1161/commits/5241972df511674286d8bc89a786b40159868b18

File: en/releases/release-1.1.12.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# TiDB Operator 1.1.12 Release Notes
+
+Release date: April 15, 2021
+
+TiDB Operator version: 1.1.12
+
+## New Features
+
+- Support setting customized environment variables for backup and restore job containers ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
+- Add additional volume and volumeMount configurations to TidbMonitor([#3855](https://github.com/pingcap/tidb-operator/pull/3855), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (ran-huang) [inline en/releases/release-1.1.12.md:14]:
```suggestion
- Add additional volume and volumeMount configurations to TidbMonitor ([#3855](https://github.com/pingcap/tidb-operator/pull/3855), [@mikechengwei](https://github.com/mikechengwei))
```

File: en/releases/release-1.1.12.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- The resources in the tidb-operator chart use the new service account when `appendReleaseSuffix` is set to `true` ([#3819](https://github.com/pingcap/tidb-operator/pull/3819), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Support setting customized store labels according to the node labels ([#3784](https://github.com/pingcap/tidb-operator/pull/3784), [@L3T](https://github.com/L3T))
+
+## Improvements
+
+- Add retry for DNS lookup failure exception in TiDBInitializer ([#3884](https://github.com/pingcap/tidb-operator/pull/3884), [@handlerww](https://github.com/handlerww))
+- Support multiple PVCs for PD during scaling and failover ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
+- Optimize `PodsAreChanged` function ([#3901](https://github.com/pingcap/tidb-operator/pull/3901), [@shonge](https://github.com/shonge))
```
Comment (ran-huang) [inline en/releases/release-1.1.12.md:22]:
```suggestion
- Optimize the `PodsAreChanged` function ([#3901](https://github.com/pingcap/tidb-operator/pull/3901), [@shonge](https://github.com/shonge))
```

File: en/releases/release-1.1.12.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- Add E2E serial test case "Canary Deploy Operator from 1.1.10 to latest" ([#3764](https://github.com/pingcap/tidb-operator/pull/3764), [@shonge](https://github.com/shonge))
```
Comment (ran-huang) [inline en/releases/release-1.1.12.md:23]:
```suggestion
```
Users don't need to know about tests.

File: zh/releases/release-1.1.12.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- 支持备份恢复 CR 设置 affinity 和 tolerations ([#3835](https://github.com/pingcap/tidb-operator/pull/3835)，[@dragonly](https://github.com/dragonly))
+- 设置 `appendReleaseSuffix` 为 `true` 时，支持 tidb-operator chart 使用新的 service account ([#3819](https://github.com/pingcap/tidb-operator/pull/3819)，[@DanielZhangQD](https://github.com/DanielZhangQD))
+- 支持设置自定义 Store 标签并根据 Node 标签获取值  ([#3784](https://github.com/pingcap/tidb-operator/pull/3784)， [@L3T](https://github.com/L3T))
+
+## 优化提升
+
+- TiDBInitializer 中增加重试机制，解决 DNS 查询异常处理问题 ([#3884](https://github.com/pingcap/tidb-operator/pull/3884)，[@handlerww](https://github.com/handlerww))
+- 在 PD 的扩缩容和容灾过程中增加多 PVC 支持 ([#3820](https://github.com/pingcap/tidb-operator/pull/3820)，[@dragonly](https://github.com/dragonly))
+- 优化 `PodsAreChanged` 函数 ([#3901](https://github.com/pingcap/tidb-operator/pull/3901), [@shonge](https://github.com/shonge))
+- 增加 "Canary Deploy Operator from 1.1.10 to latest" E2E serial 测试用例 ([#3764](https://github.com/pingcap/tidb-operator/pull/3764), [@shonge](https://github.com/shonge))
```
Comment (ran-huang) [inline zh/releases/release-1.1.12.md:23]:
```suggestion
```
ditto

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Comment by ran-huang** [comment]
@DanielZhangQD Is this mergeable?

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**